app/src/main/kotlin/com/flxrs/dankchat/main/MainFragment.kt

```diff
@@ -713,6 +714,7 @@ class MainFragment : Fragment() {
                 scrollBackLengthKey -> mainViewModel.setScrollbackLength(ChatSettingsFragment.correctScrollbackLength(p.getInt(scrollBackLengthKey, 10)))
                 keepScreenOnKey -> keepScreenOn(p.getBoolean(key, true))
                 suggestionsKey -> binding.input.setSuggestionAdapter(p.getBoolean(key, true), suggestionAdapter)
+                preferEmotesSuggestionsKey -> mainViewModel.preferEmoteSuggestions.value = p.getBoolean(key, false)
```

Comment:
As I said in the other comment, use a function from the viewmodel to change the mutable flow here